### PR TITLE
Removed the hard-coded /wiki and send wiki-data when type is wiki

### PR DESCRIPTION
**Pull request by broekhuisg**
Heb de /wiki folder eruit gesloopt en heb in de getServersideprops aangegeven:

als wikipage > stuur alle wikipages met title/url mee om het menu aan de zijkant te genereren.

### CODE REVIEW

**Review by mattijsstam — CHANGES_REQUESTED** [review]
Apart from some small issues the code is approved, there are however two major issues:

1. There are no tests for this functionality
2. The breadcrumbs component is not updated to match the new url structure and therefore does not work anymore

**Review by antw — APPROVED** [review]
The changes look fine to me, except for the issues raised by @mattijsstam.

There are a bunch of TypeScript warnings in [frontend/pages/[...path].tsx](https://github.com/ZEnMo/Holon-webapp/pull/116/files#diff-fa6bcaceb5ae401380bacd6775f686e78d91c211c916f64076a3f90127997f8f), but none of those seems to have been introduced by this PR.

I'm not actually able to run this for myself in the dev container. I followed the instructions in the readme, and both the front- and back-end start fine, but navigating to the wiki results in an error:

```js
// Unhandled Runtime Error
// WagtailApiResponseError: Not Found. Url: /wt/api/nextjs/v1/external_view_data/404/. Params: {}
// 
// Source
// api/wagtail.ts (90:22) @ eval
// 
  88 | 
  89 | if (res.status < 200 || res.status >= 300) {
> 90 |     const error = new WagtailApiResponseError(res, url, params);
     |                  ^
  91 |     error.response = res;
  92 |     throw error;
  93 | }
```

Am I doing something wrong, or are there additional steps to take? My instinct is that the page should be defined in the (Wagtail) database, and this means either Next or Wagtail can't find them. Maybe I'm missing a step for creating the pages?

**Comment by broekhuisg** [comment]
@antw Did you create a Wikipage first in the admin? When you navigate to localhost:8000/wt/cms and login with admin/admin, you can create a Wikipage by clicking on: Pages > Home Page > 3 dots at the top > Add Child Page. If you add a page called 'testpage' for example, you can navigate to localhost:3000/testpage.

If have done this, does it work?

**Comment by antw** [comment]
> If have done this, does it work?

That worked, thanks! It works fine.

I've not used Wagtail before and the readme mentions visiting `/admin`, but that didn't work (404). I eventually found `/wt/admin` but that didn't have the option to create a page. Could we add something about this to the readme?

File: frontend/containers/WikiPage/WikiPage.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ const WikiPage = ({ richText, allPages }: WikiContainerProps) => {
 
         <main className="flex flex-1 flex-col">
           <div className="border-b-2 border-gray-200 py-3 pl-10">
-            {allPages.items && <Breadcrumbs path={path} posts={pages} />}
+            {pages && <Breadcrumbs path={path} posts={pages} />}
```
[the remark this answers is not in the document]
Comment (mattijsstam) [reply frontend/containers/WikiPage/WikiPage.tsx:77]: moved to #122

File: frontend/pages/[...path].tsx [hunk 2]
```diff
@@ -1,128 +1,121 @@
...
+          permanent: isPermanent,
+        },
+      };
+    }
 
-    const { host } = req.headers;
-    let queryParams = new URL(req.url, `https://${host}`).search;
-    if (queryParams.indexOf('?') === 0) {
-        queryParams = queryParams.substr(1);
+    if (componentName === "WikiPage") {
```
[the remark this answers is not in the document]
Comment (mattijsstam) [reply frontend/pages/[...path].tsx:65]: moved to #122

**Review by mattijsstam — APPROVED** [review]

**Pull request merged**